design/RELEASE_CADENCE.md

```diff
@@ -0,0 +1,67 @@
+# MetalLB release cadence
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (Optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Summary
+
+The purpouse of this enhancement is to define a versioning and release schedule tied to
+kubernetes versions.
+
+## Motivation
+
+The reason for this proposal is to establish a dependency between a given MetalLB version and kubernetes versions.
+
+This will make MetalLB more reliable with regards with the set of kubernetes apis it can use or can't use. A good example for it is the introduction of the use of EndpointSlices, which are versioned as beta in k8s 1.17+ but moving to stable in 1.21.
```

Comment:
Sorry, but I don't follow. What would this proposal simplify, for example with endpoint slices?